### PR TITLE
New Source: WuxiaWorld

### PR DESCRIPTION
**Pull request by Nabeelshar**
Works fine with novel but doesn't get manga.
Example:-
https://wuxiaworld.site/novel/national-school-prince-is-a-girl-comics-wuxia-dao-novel/
https://wuxiaworld.site/novel/national-school-prince-is-a-girl-comics-wuxia-dao-novel/chapter-79-identity-revealed/

### CODE REVIEW

File: app/src/main/java/my/noveldokusha/scraper/WuxiaWorld.kt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+//    override suspend fun getChapterText(doc: Document): String
+//    {
+//        return doc.selectFirst(".reading-content p")!!.let { textExtractor.get(it) }
+//    }
+
+    override suspend fun getBookCoverImageUrl(doc: Document): String?
+    {
+        return doc.selectFirst("div.summary_image")
+            ?.selectFirst("img")
+            ?.attr("data-src")
```
Comment (Nabeelshar) [inline app/src/main/java/my/noveldokusha/scraper/WuxiaWorld.kt:26]: need to change the base curl it get from https://wuxiaworld.b-cdn.net/ to https://wuxiaworld.site/

**Comment by nanihadesuka** [comment]
> Works fine with novel but doesn't get manga. Example:- https://wuxiaworld.site/novel/national-school-prince-is-a-girl-comics-wuxia-dao-novel/ https://wuxiaworld.site/novel/national-school-prince-is-a-girl-comics-wuxia-dao-novel/chapter-79-identity-revealed/

This app is novels only, for manga use Tachiyomi.

**Comment by Nabeelshar** [comment]
This is a novel source as a side it has novel comic. (Comics of webnovels which are already translated.)

**Comment by nanihadesuka** [comment]
@Nabeelshar the Source interface has changed with commit ca745072c2685a92e11105dce73332b1ef5a8dab.
Needs changes, then will review :)

**Comment by Nabeelshar** [comment]
also fixed  novel image not showing.

**Pull request merged**